plugins/modules/ntnx_pc_restorable_domain_managers_info_v2.py

```diff
@@ -155,6 +155,22 @@ def get_restorable_domain_managers(module, domain_manager_backups_api, result):
             msg="Api Exception raised while fetching restorable domain managers info",
         )
 
+    total_available_results = (
+        strip_internal_attributes(resp.to_dict())
+        .get("metadata")
+        .get("total_available_results")
+    )
+
+    if total_available_results == 0:
```

Comment:
Shouldnt fail in this case. We just put it in result.total_available_results = 0

Reply:
Add in docs about this new field in result